pkgs/development/libraries/librealsense/default.nix

```diff
@@ -42,22 +42,9 @@ stdenv.mkDerivation rec {
     ++ lib.optionals enableGUI [ mesa gtk3 glfw libGLU curl ];
 
   patches = [
-    # fix build on aarch64-darwin
-    # https://github.com/IntelRealSense/librealsense/pull/9253
-    (fetchpatch {
-      url = "https://github.com/IntelRealSense/librealsense/commit/beb4c44debc8336de991c983274cad841eb5c323.patch";
-      sha256 = "05mxsd2pz3xrvywdqyxkwdvxx8hjfxzcgl51897avz4v2j89pyq8";
-    })
-    ./py_sitepackage_dir.patch
     ./py_pybind11_no_external_download.patch
   ];
 
```

Comment:
```suggestion

  preInstall = ''
    export HOME="$(mktemp -d)"
  '';

```

Reply:
or if possible, put the presets in `$out/share/librealsense`

Reply:
Thanks for the review, fixed